### PR TITLE
New Hai/Wanderer missions

### PR DESCRIPTION
**Pull request by jafdy**
Story line that ends with "event: Language: Wanderer direct" to
represent your speaking machine being able to translate directly to the
wanderer language.

### CODE REVIEW

File: data/hai missions.txt [hunk 1]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+	landing
+	source "Vara K'chrai"
+	destination Earth
+	to offer
+		has "Language: Wanderer 1B: done"
+	on offer
+		event "eruk collects wanderer data" 366
+		conversation
+			`When you are coming into landing on Vara K'chrai, you radio ahead and ask Sayari, the Hai ambassador, and Iktat Ret, the Wanderer envoy, to meet you when you land.`
+			`	When you step out of your ship you find a larger group of Wanderers, some armed, than you where you expecting. Ret comes hurrying up behind them. "Sorry <first>, one of the ships in orbit detectid a Hai [stowaway, passenger} on-board your ship. We..." Ret stops talking when Eruk appears behind you.`
```
Comment (Gkramme) [inline data/hai missions.txt]: [detected, found] and after passenger there should be a square bracket

File: data/hai missions.txt [hunk 2]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+				`	(Wait it out)`
+					goto wait
+				`	"Can't you make the decision?"`
+
+			`	"No, I only have [authority, command] concerning Human [visiters, raiders]. They came to deal with any Hai so they make the decision." It looks like you have no choice but to wait for them to finish.`
+
+			label wait
+			`	When the hubub has quieted down a little one of the unarmed Wanderers steps out of the croud and proclames (rather grandly) "Hai Eruk and Human <first>, you are to go before the Assembly to determine the validity of Hai Eruk's request of the Wanderers. Hai Eruk you will be kept under guard until such time it is deemed unnecessary." And with that most of the Wanderers fly off leaving just Ret, the two armed guards, Sayari and Eruk on your ship's landing pad.`
+			`	"What where you thinking!" Sayari says rounding on Eruk.`
+			`	"I needed someone to get me here and I  knew you woulden't agree to it so I asked Captin <last> to bring me here."`
```
Comment (Gkramme) [inline data/hai missions.txt]: Captian

**Comment by Gkramme** [comment]
This is missing a lot of commas. Can I help you with this?

File: data/hai missions.txt [hunk 3]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+					goto assembly
+				`	(try to comfort him)`
+
+			`	You hurry after him but when you catch up he simply angrily waves you away. You resign yourself to simply having to follow him.`
+
+			label assembly
+			`	When you reach the Assembly building you notice large numbers of Wanderers flying into it to through an open roof but you can't help but notice that there are no visible doors on the side of the building.`
+			`	Eruk sighs and goes to sit down on a nearby wall while you start to walk around the edge of the building looking for a door.`
+			`	After going nearly the entire way round the building you find Iktat Ret and another slightly smaller Wanderer standings by the only door you've seen so far. Eruk is also with them but he doesn't seem to be any happier than before.`
+			`	"<first>, this is my child Iktat Kri. [They?, He?, She?] will be able to [understand, interpret] for you."`
```
Comment (Gkramme) [inline data/hai missions.txt]: The presence of they doesn't make any sense here

File: data/hai missions.txt [hunk 4]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+			`	You realise that you've never seen a female Wanderer but maybe Wanderer society does not differentiate between male and female in the same way Hai and Human societies do, and the opposite of what is done in the Syndicate.`
+			`	Iktat Ret wishes Eruk luck before indicating that you should follow Iktat Kri inside.`
+			`	Inside the building looks even more like a bird's nest with hundreds of balconies scattered chaotically around the inside of the large bowl-shaped structure. All of the balconies are full apart from a few at the very top of the bowl but even these are filled quickly as the last Wanderers fly in.`
+			`	The half-dozen or so lowest balconies are filled with some very old looking Wanderers, this1 must be the actual Wanderer government as you can't help but feel that any debate between the number of Wanderer that are currently occupying the building would take so long that it would be completely pointless.`
```
Comment (Gkramme) [inline data/hai missions.txt]: This1 is a typo

File: data/hai missions.txt [hunk 5]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+			`	Once all the Wanderers have settled down and the last rustling of wings has ceased, one of the old Wanderers sitting in front of you leans forward and asks Eruk, with Iktat Kri translating, to tell them about his proposition.`
+			`	Eruk once again explains what he wants to do and how he needs the Wanderers help to do it.`
+			`	When he's finished the wanderer turns to you and asks "<first>, what is your [thought, opinion] on this."`
```
Comment (Gkramme) [inline data/hai missions.txt]: Question mark is needed here

**Comment by jafdy** [comment]
Go for it. Story writing is not my strong point. I'm better at setting the missions up.

**Comment by Gkramme** [comment]
Can you grant me write acess?

File: data/hai missions.txt (hunk 3 above)
Comment (jafdy) [inline data/hai missions.txt]: The point is that the wanderers don't have words for male/female so it won't translate well. Eruk mentions this if you visit him. I could take it out and have [He?, She?] instead.

**Comment by jafdy** [comment]
Just be careful what you touch.

**Comment by Gkramme** [comment]
Ok

**Comment by jafdy** [comment]
When the original missions where written the called the translator a "speaking machine" so I was continuing with that. Some of the imperfect translations where the exact same as the one in the original missions so I kept them as they where.

File: data/hai missions.txt [hunk 6]
```diff
@@ -862,7 +862,7 @@ mission "Language: Wanderer 2B"
 	on complete
 		payment 1000000
 		conversation
-			`Eruk seems to be a bit rattled from the attacks you had on your way here. "I can't think who would want me dead for the work I've been doing. Hopefully they won't be able to find me, now you've got me safely here." He looks very worried. "I should have this software sorted in a couple of months, so when I do I'll send a message telling you how to update. Also, here's a bonus for keeping me alive." he gives you a wink and hands you a 1 million credit chip.`
```
Comment (Amazinite) [inline data/hai missions.txt]: Rip wink

File: data/hai missions.txt [hunk 7]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+	blocked "You need 11 tonnes of cargo space and a free bunk to accept this mission. Come back when you have made some more room."
+	source Greenwater
+	destination "Vara K'chrai"
+	cargo "Eruks equipment" 11
+	passengers 1
+	to offer
+		has "Language: Wanderer 1A: done"
+	on offer
+		conversation
+			`As soon as you open the hatch you get knocked over as Eruk rushes into your ship. He seems to want to get going as soon as posable. You also notice he hasn't got any luggage.`
```
Comment (toilethinges) [inline data/hai missions.txt]: Possible.

File: data/hai missions.txt [hunk 8]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+			label hit
+			`	"Forgot everything" he says, rushing out and knocking you over again as he goes past.`
+				goto return
+
+			label forgot
+			`	He turns around and you narrowly avoid getting hit again as he rushes out.`
+				goto return
+
+			label remind
+			`	"Yes, I have," he says turning around and nearly hitting again as he rushes out.`
```
Comment (toilethinges) [inline data/hai missions.txt]: hitting you*

File: data/hai missions.txt [hunk 9]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+
+			label explain
+			`	He tells the Wanderers everything he told you and more. It seems he has thought of everything from data sets to time frames and processing power to memory space. When he finishes, the wanderers start to talk among themselves in their own chirping, clicking language getting louder and faster all the time.`
+			`	Ret moves towards you, Sayari and Eruk. "They can't [decide, agree] what to do, some think we should [stop, decline] his request and send him back to Hai space and others think we should [support?, care?, help?] him outright, the rest have every view inbetween."`
+			choice
+				`	(Wait it out)`
+					goto wait
+				`	"Can't you make the decision?"`
+
+			`	"No, I only have [authority, command] concerning Human [visiters, travelers]. They came to deal with any Hai so they make the decision." It looks like you have no choice but to wait for them to finish.`
```
Comment (toilethinges) [inline data/hai missions.txt]: Visitors.

File: data/hai missions.txt [hunk 10]
```diff
@@ -877,7 +877,7 @@ mission "Language: Wanderer 2B attacked"
 			branch with
 				has "Language: Wanderer 2B: active"
 
-			`The ship carrying Eruk, the Kach'cha takes off immediately before you. Coming out of the planets atmosphere you see it getting ready to jump, you also see the Redacted coming up fast behind it. At close range it opens fire with weapons much more powerful then a ship it size has any right to possess, the Kach'cha is disabled almost immediately. The "Redacted" then does something you have never seen any ship do before, it proceeds to ram the Kach'cha, utterly destroying both ships in the process.`
+			`The ship carrying Eruk, the Kach'cha takes off immediately before you. Coming out of the planets atmosphere you see it getting ready to jump, you also see the Redacted coming up fast behind it. At close range it opens fire with weapons much more powerful then a ship it size has any right to possess, the Kach'cha is disabled almost immediately. The Redacted then does something you have never seen any ship do before, it proceeds to ram the Kach'cha, utterly destroying both ships in the process.`
```
Comment (toilethinges) [inline data/hai missions.txt]: Should probably replace the comma with a colon (, - :) after the phrase "The Redacted then does something you have never seen any ship do before,"

File: data/hai missions.txt [hunk 11]
```diff
@@ -509,7 +509,7 @@ mission "Language: Wanderer 1B"
 		has "Language: Wanderer 1A: done"
 	on offer
 		conversation
-			`As soon as you open the hatch you get knocked over as Eruk rushes into your ship. He seems to want to get going as soon as posable. You also notice he hasn't got any luggage.`
+			`As soon as you open the hatch you get knocked over as Eruk rushes into your ship. He seems to want to get going as soon as posible. You also notice he hasn't got any luggage.`
```
Comment (Amazinite) [inline data/hai missions.txt]: possible, not posible

**Comment by Gkramme** [comment]
@endless-sky any problems here?

File: data/hai missions.txt [hunk 12]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+			choice
+				`	"Ow, my head."`
+					goto hit
+				`	"Traveling light?"`
+					goto forgot
+				`	"I think you have forgotten your luggage."`
+					goto remind
+
+			label hit
+			`	"Forgot everything" he says, rushing out and knocking you over again as he goes past.`
```
Comment (toilethinges) [inline data/hai missions.txt]: "everything" -> "everything,"

File: data/hai missions.txt [hunk 13]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+
+
+mission "Language: Wanderer end"
+	landing
+	invisible
+	to offer
+		has "event: language: Wanderer direct"
+	on offer
+		conversation
+			`You've received a message from Eruk, "<first>, I've done it. If you load up the attached file onto your speaking machine it will update it to be able to directly translate into the Wanderer language." Unfortunately he appears to have forgotten to attach the file.`
```
Comment (toilethinges) [inline data/hai missions.txt]: "Unfortunately" - > "Unfortunately,"

File: data/hai missions.txt [hunk 14]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+			`	When you step out of your ship you find a larger group of Wanderers, some armed, than you where you expecting. Ret comes hurrying up behind them. "Sorry <first>, one of the ships in orbit detectid a Hai [stowaway, passenger] on-board your ship. We..." Ret stops talking when Eruk appears behind you.`
+			choice
+				`	(Let Eruk explain)`
+					goto explain
+				`	(Try to explain)`
+
+			`	"Eruk had an idea, he wants to improve his speaking machine to handle your language." Eruk stops you and starts explaining himself.`
+
+			label explain
+			`	He tells the Wanderers everything he told you and more. It seems he has thought of everything from data sets to time frames and processing power to memory space. When he finishes, the wanderers start to talk among themselves in their own chirping, clicking language getting louder and faster all the time.`
```
Comment (toilethinges) [inline data/hai missions.txt]: Wanderers need to be capitalized (check the whole document, preferably).

File: data/hai missions.txt [hunk 15]
```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
...
+		government Unidentified
+		system "Lom Tahr"
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	on visit "Some of the unidentified ships are still in orbit. You will have to take off again to try and lose them."
+
+	on complete
+		payment 1000000
+		conversation
+			`Eruk seems to be a bit rattled from the attacks you had on your way here. "I can't think who would want me dead for the work I've been doing. Hopefully they won't be able to find me, now you've got me safely here." He looks very worried. "I should have this software sorted in a couple of months, so when I do I'll send a message telling you how to update. Also, here's a bit of a bonus for keeping me alive." he hands you a 1 million credit chip. "If I ever need you again, I'll give you a call." and with that, he walks off.'`
```
Comment (toilethinges) [inline data/hai missions.txt]: Hopefully, they won't (or: "now that you've got me safely here")

He needs to be capitalized.

So when I do, I'll

File: data/hai missions.txt (hunk 12 above)
Comment (jafdy) [inline data/hai missions.txt]: Your confused.

File: data/hai missions.txt (hunk 13 above)
Comment (jafdy) [inline data/hai missions.txt]: Very confused.

File: data/hai missions.txt (hunk 12 above)
Comment (Alkallid) [inline data/hai missions.txt]: The comma is needed.

**Comment by jafdy** [comment]
Giving this up I think.

**Pull request closed without merging**